features/test_error.feature

```diff
@@ -6,6 +6,8 @@ Feature: Error test trial
         Given the application has been launched
         And I start a new application
         And I select Standard rules as the permit type
+        And I select Car and vehicle dismantling as the permit category
+        And I select SR2015 No 13 as the permit number
         And I select none as the permit holder
```

Comment:
Should this be `And I enter my permit holder details for none` ?

Reply:
Changed as requested